network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_full_width_image_block.html

```diff
@@ -24,9 +24,10 @@
     </figure>
   </div>
 
-{% if not is_last %}
-  {% comment %} Restart the Article Block bootstrap grid {% endcomment %}
-  <div class="container">
-    <div class="row">
-      <div class="offset-sm-1 col-sm-10 col-12 article-blocks">
-{% endif %}
+  {% comment %} Restart the Article Block bootstrap grid. If it is the last item, hide the empty container to keep from pushing footer down a couple px{% endcomment %}
```

Comment:
Hi @KalobTaulien @mmmavis,

I believe that the 3 mismatched divs were coming from **article_page.html**, as there are 3 closing divs on lines 45,46,47, that are expecting to close the three container divs on lines 16,17,18.

After finding that the error was only reproduced when there was a wide image block at the end of the article page, I then found that the **wide_image_block** file starts by closing the 3 container divs from **article_page.html** on lines 4,5,6, but then does **not** reopen them if the image block is the last item in the for loop.

To solve the issue, I removed the if statement so the image block can open the three container divs **article_page.html** is expecting to close, which removes the error. 

However, this did create a empty container object that pushed the contents below down just a bit.

To alleviate this, I then added a if statement in the class, which checks if the image block is the last item in the for loop. If so, it adds the "hidden" class to remove the extra container div to avoid anything else being pushed down in the page, which I confirmed with percy. 

If you have any questions feel free to let me know and I can explain it through a zoom call :) Thank you!